docs/galaxy/docsite/README.md

```diff
@@ -0,0 +1,48 @@
+Homepage and Documentation Source for Ansible Galaxy
+====================================================
+
+This project hosts the source behind [docs.ansible.com/galaxy](https://docs.ansible.com/galaxy)
+
+Contributions to the documentation are welcome. To make changes, submit a pull request that changes the reStructuredText files in the `rst/` directory only, and the core team can do a docs build and push the static files.
+
+If you wish to verify output from the markup such as link references, you may install sphinx and build the documentation by running `make webdocs` from the `ansible/docsgalaxy/docsite` directory.
+
```

Comment:
Actually, we could just link to the page on docs.ansible.com, and not have to maintain this stuff in multiple places.